code/game/gamemodes/cult/talisman.dm

```diff
@@ -278,11 +259,131 @@ Rite of Disorientation
+			H.reagents.add_reagent("mindbreaker", 30)
+		qdel(src)
+
+//Talisman of Fabrication: Creates a construct shell out of 25 metal sheets.
+/obj/item/weapon/paper/talisman/construction
+	cultist_name = "Talisman of Construction"
+	cultist_desc = "Use this talisman on at least twenty-five metal sheets to create an empty construct shell"
+	invocation = "Ethra p'ni dedol!"
+	color = "#000000" // black
+
+/obj/item/weapon/paper/talisman/construction/attack_self(mob/living/user)
+	if(iscultist(user))
+		user << "<span class='warning'>To use this talisman, place it upon a stack of metal sheets.</span>"
+	else
+		user << "<span class='danger'>There are indecipherable images scrawled on the paper in what looks to be... <i>blood?</i></span>"
+
+
+/obj/item/weapon/paper/talisman/construction/attack(obj/M,mob/living/user)
+	if(iscultist(user))
+		user << "<span class='cultitalic'>This talisman will only work on a stack of metal sheets!</span>"
+		log_game("Construct talisman failed - not a valid target")
+
+/obj/item/weapon/restraints/handcuffs/energy/cult //For the talisman of shackling
+	name = "cult shackles"
+	desc = "shackles that bind the wrists with sinister magic."
+	trashtype = /obj/item/weapon/restraints/handcuffs/energy/used
+	origin_tech = "materials=2;magnets=5"
+
+/obj/item/weapon/restraints/handcuffs/energy/cult/used
+	desc = "magical remnants"
+
+/obj/item/weapon/restraints/handcuffs/energy/cult/used/dropped(mob/user)
+	user.visible_message("<span class='danger'>Your [src] shatter in a discharge of dark magic!</span>", \
+							"<span class='userdanger'>[user]'s [src] shatters in a discharge of dark magic!</span>")
+	var/datum/effect_system/spark_spread/S = new
+	S.set_up(4,0,user.loc)
+	S.start()
+	qdel(src)
+
+//Talisman of Shackling: Applies special cuffs directly from the talisman
+/obj/item/weapon/paper/talisman/shackle
+	cultist_name = "Talisman of Shackling"
+	cultist_desc = "Use this talisman on a victim to handcuff them with dark bindings."
+	invocation = "In'totum Lig'abis!"
+	color = "#B27300" // burnt-orange
+	uses = 4
+
+/obj/item/weapon/paper/talisman/shackle/invoke(mob/living/user, successfuluse = 0)
+	if(successfuluse) //if we're forced to be successful(we normally aren't) then do the normal stuff
+		return ..()
+	if(iscultist(user))
+		user << "<span class='warning'>To use this talisman, attack the target directly.</span>"
+	else
+		user << "<span class='danger'>There are indecipherable images scrawled on the paper in what looks to be... <i>blood?</i></span>"
+	return 0
+
+/obj/item/weapon/paper/talisman/shackle/attack(mob/living/target, mob/living/user)
+	if(iscultist(user))
+		if(isrobot(target))
+			..()
+			return
+		if(!isliving(target))
+			user.visible_message("<span class='cultitalic'>This talisman's magic does not affect the dead!</span>")
+			return
+		var/mob/living/L = target
+		if(ishuman(L))
+			CuffAttack(L,user)
+
+/obj/item/weapon/paper/talisman/shackle/proc/CuffAttack(mob/living/L,mob/living/user)
```

Comment:
Last time I went down this road it triggered a long series of errors. Is there any practical reason for changing it from its current state?